src/__tests__/NavBar.test.js

```diff
@@ -0,0 +1,11 @@
+import '@testing-library/jest-dom';
+import { render } from '@testing-library/react';
+import NavBar from '../Components/NavBar';
+
+test('renders the navbar component', () => {
+  const { asFragment } = render(
+    <NavBar />,
+  );
+
+  expect(asFragment()).toMatchSnapshot();
+});
```

Comment:
- [ ] Let's add tests for some pure functions in Redux code as requested in project requirements